test/tests/inittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This have to be first test for cockpit
+#  install build dependencies
+#  download new version of cockpit from git 
+#  compile it
+# it is basic test compilation must PASS
+
+set -x
+
+BUILD="build1"
+PACKAGE="cockpit"
+SOURCE="https://github.com/$PACKAGE-project/$PACKAGE.git"
+
+#
+yum-builddep -y $PACKAGE
+yum-builddep -y $PACKAGE
+
+ /bin/rm -fr $PACKAGE
+ git clone $SOURCE && \
```

Comment:
I think you shouldn't be cloning the source from github here.  Instead, we want to compile the same sources that contain inittest.sh.

(Or alternatively, the tests should come from what you clone here, not from the outside.)